translations/base-zh-TW.yaml

```diff
@@ -809,10 +809,10 @@ keybindings:
         comparator: 比對機
         item_producer: 物品生產機（沙盒模式）
         copyWireValue: 電路：複製數值於游標底下
-        rotateToUp: "Rotate: Point Up"
-        rotateToDown: "Rotate: Point Down"
-        rotateToRight: "Rotate: Point Right"
-        rotateToLeft: "Rotate: Point Left"
+        rotateToUp: 轉動: 向上
+        rotateToDown: 轉動: 向下
+        rotateToRight: 轉動: 向右
+        rotateToLeft: 轉動: 向左
```

Comment:
There's a syntax error on these lines - colons are special characters, you have to keep the quotes